SUMMARY.md

```diff
@@ -35,6 +26,19 @@
   * [📊 Collateral Ratio](concepts/other-aspects/collateral-ratio.md)
   * [🚑 Risks and Emergency Modules](concepts/other-aspects/emergency-modules.md)
   * [⌛ Collateral Settlement](concepts/other-aspects/collateral-settlement.md)
+* [📔 App Guides](app-guides/README.md)
```

Comment:
Not sure all the guides are relevant in this subsection: like first steps in crypto and get agEUR are relevant cross-module

Same for earning ANGLE (you can check my PR for this)

Reply:
Yep, true for first steps and agEUR
The earning ANGLE links to the page in the DAO guides section, I put it here just as additional reference

Reply:
Let's make sure we don't have the same reference twice